deliver/lib/deliver/submit_for_review.rb

```diff
@@ -73,17 +74,25 @@ def wait_for_build(app)
         end
 
         latest_build = find_build(candidate_builds)
+        latest_build_matches = (latest_build && latest_build.train_version == app_version && latest_build.build_version == build.build_version)
```

Comment:
You can use `&.`.

```suggestion
        latest_build_matches = (latest_build&.train_version == app_version && latest_build&.build_version == build&.build_version)
```

Reply:
This line just extracts the expression from the line below, but what you proposed is a great syntax I didn't know about!

Reply:
I'm sorry, this syntax is not allowed on fastlane project 😢 
`&.` was introduced since Ruby 2.3, but unfortunately, TargetRubyVersion is still 2.0.
https://github.com/fastlane/fastlane/blob/master/.rubocop.yml#L244

This syntax is correct, but we can't use this.
https://circleci.com/gh/fastlane/fastlane/30932?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
`build` seems to be optional, so we have to do nil-cheking for `build`.

`latest_build_matches = (latest_build && build && latest_build.train_version == app_version && latest_build.build_version == build.build_version)`